Library/Formula/supervisor.rb

```diff
@@ -0,0 +1,108 @@
+class Supervisor < Formula
+  desc "Process Control System"
+  homepage "http://supervisord.org/"
+  url "https://github.com/Supervisor/supervisor/archive/3.2.1.tar.gz"
+  sha256 "d9c0b17ec42ac6477e7ef0ad2e1f6f1597f855a4d9606c024ac40eb034d6e9ed"
+
+  resource "meld3" do
+    url "https://pypi.python.org/packages/source/m/meld3/meld3-1.0.2.tar.gz"
+    sha256 "f7b754a0fde7a4429b2ebe49409db240b5699385a572501bb0d5627d299f9558"
+  end
+
+  def install
+    # Before the example Supervisor configuration file gets installed, modify
+    # it to use Homebrew's conventions.
+    inreplace buildpath/"supervisor/skel/sample.conf" do |s|
+      # A `[unix_http_server]` modification
+      s.gsub! %r{(?<=^file=)/tmp/supervisor.sock}, var/"run/supervisor.sock"
+
+      # Some `[supervisord]` modifications
+      s.gsub! %r{(?<=^logfile=)/tmp/supervisord.log}, var/"log/supervisord.log"
+      s.gsub! %r{(?<=^pidfile=)/tmp/supervisord.pid}, var/"run/supervisord.pid"
+
+      # A `[supervisorctl]` modification
+      s.gsub! %r{(?<=serverurl=)unix:///tmp/supervisor.sock}, "unix://#{var}/run/supervisor.sock"
+
+      # An `[include]` modification
+      s.gsub! ";files = relative/directory/*.ini",
+              "files = #{etc}/supervisor.d/*.ini"
+    end
+
+    etc.install buildpath/"supervisor/skel/sample.conf" => "supervisord.ini"
+
+    resource("meld3").stage do
+      system "python", *Language::Python.setup_install_args(libexec/"vendor")
+    end
+
+    system "python", *Language::Python.setup_install_args(libexec)
+
+    bin.install Dir[libexec/"bin/*"]
+
+    ENV.prepend_create_path "PYTHONPATH", libexec/"vendor/lib/python2.7/site-packages"
```

Comment:
Yeah, these `PYTHONPATH` lines are so far down because I was exploring what was actually *required* for this formula for build correctly. So, while I think it's notable that the resource install and main compile succeeded without the `PYTHONPATH` environment tweaks, I'm happy to defer to the Homebrew conventions.